### ISSUE TITLE
published `types` is incorrect

### ISSUE THREAD

**Issue by JounQin** [angular-eslint/angular-eslint#479]
**Description and reproduction of the issue**

https://unpkg.com/browse/@angular-eslint/template-parser@12.0.0/dist/index.d.ts

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint       `        | `X.Y.Z` |
| `TypeScript`                       | `X.Y.Z` |
| `ESLint`                           | `X.Y.Z` |
| `node`                             | `X.Y.Z` |

```sh
# Please run `npx ng version` in your project and paste the full output here:


```

- [ ] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by JamesHenry** [comment]
Good catch, thanks @JounQin!
